Assets/HoloToolkit/Utilities/Scripts/Editor/EnforceEditorSettings.cs

```diff
@@ -11,23 +10,68 @@ namespace HoloToolKit.Unity
     [InitializeOnLoad]
     public class EnforceEditorSettings
     {
+        private const string _assemblyReloadTimestampKey = "HoloToolkit_Editor_LastAssemblyReload";
+
         static EnforceEditorSettings()
         {
             #region Editor Settings
 
-            if (EditorSettings.serializationMode != SerializationMode.ForceText)
-            {
-                EditorSettings.serializationMode = SerializationMode.ForceText;
-                UnityEngine.Debug.Log("Setting Force Text Serialization");
-            }
+            if (TrueOnce()) {
```

Comment:
Rather than indent the entire method, it may be cleaner to check the opposite condition and return early.